src/components/ConfirmationModal.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const ConfirmationModal = ({ isOpen, onConfirm, onCancel }) => {
```

Comment:
This makes more sense since we are using this modal only for delete action

```suggestion
const DeleteConfirmationModal = ({ isOpen, onConfirm, onCancel }) => {
```